install_config/configuring_pipeline_execution.adoc

```diff
@@ -65,12 +65,22 @@ jenkinsPipelineConfig:
 <5> Optional values to pass to the template during instantiation.
 ====
 
-[IMPORTANT]
-====
-When a Pipeline build configuration is created,
-{product-title} instantiates the `*jenkinsPipelineConfig*` template *only if*
-no existing service name in the project matches the `*serviceName*` field.
-This means `*serviceName*` must be chosen such that it is unique in the project.
+When a Pipeline build configuration is created, {product-title} looks for a
+Service matching `*serviceName*`. This means `*serviceName*` must be chosen
+such that it is unique in the project. If no Service is found, {product-title}
+instantiates the `*jenkinsPipelineConfig*` template. If this is not desirable
+(if you would like to use a Jenkins server external to {product-title}, for
+example), there are a few things you can do, depending on who you are.
+
+* If you're a cluster admin, simply set `*autoProvisionEnabled*` to `*false*`.
+  This will disable autoprovisioning across the cluster.
+* If you're an unpriviledged user, a Service must be created for {product-title}
+  to use. The default value of `*serviceName*` is `*jenkins*`. However, this only
```

Comment:
"The service name must match the cluster configuration value of serviceName in the jenkinsPipelineConfig.  The default value is jenkins."